users/serializers.py

```diff
@@ -7,7 +7,23 @@ class Meta:
         fields = ['id', 'username', 'email', 'password', 'role']
         extra_kwargs = {'password': {'write_only': True}}
 
+
+    def create(self, validated_data):
+        role = validated_data.get('role')
+        if role and role.lower() != 'employee':
+            raise serializers.ValidationError("Role must be 'employee'.")
```

Comment:
we should not raise error to provide Employee role.
You need to fix the role by hard-coded value as: role= "employee"